pkg/model/dev.go

```diff
@@ -816,21 +817,22 @@ func TranslatePodAffinity(tr *TranslationRule, name string) {
 // ToTranslationRule translates a dev struct into a translation rule
 func (dev *Dev) ToTranslationRule(main *Dev, namespace, username string, reset bool) *TranslationRule {
 	rule := &TranslationRule{
-		Container:        dev.Container,
-		ImagePullPolicy:  dev.ImagePullPolicy,
-		Environment:      dev.Environment,
-		Secrets:          dev.Secrets,
-		WorkDir:          dev.Workdir,
-		PersistentVolume: main.PersistentVolumeEnabled(),
-		Volumes:          []VolumeMount{},
-		SecurityContext:  dev.SecurityContext,
-		ServiceAccount:   dev.ServiceAccount,
-		Resources:        dev.Resources,
-		InitContainer:    dev.InitContainer,
-		Probes:           dev.Probes,
-		Lifecycle:        dev.Lifecycle,
-		NodeSelector:     dev.NodeSelector,
-		Affinity:         (*apiv1.Affinity)(dev.Affinity),
+		Container:         dev.Container,
+		ImagePullPolicy:   dev.ImagePullPolicy,
+		Environment:       dev.Environment,
+		Secrets:           dev.Secrets,
+		WorkDir:           dev.Workdir,
+		PersistentVolume:  main.PersistentVolumeEnabled(),
+		Volumes:           []VolumeMount{},
+		SecurityContext:   dev.SecurityContext,
+		ServiceAccount:    dev.ServiceAccount,
+		PriorityClassName: main.PriorityClassName,
```

Comment:
Why this would be the only property to be read from `main` instead of doing it from `dev`? What is the difference?

Reply:
The idea is to define the `PriorityClassName` only for the main container, instead of adding it to every pod in the `services` section. We could instruct the user to define it on every `service`, but I think it adds more friction to the user. Both would work anyway. 
If we go with defining it for every service, something that we are considering is support for okteto manifest templates, and that might help too

Reply:
ah, I see, that's the difference. I think it makes sense